idd/Energy+.idd.in

```diff
@@ -100940,7 +100940,9 @@ LifeCycleCost:UsePriceEscalation,
         \key ElectricitySurplusSold
         \key ElectricityNet
         \key NaturalGas
-        \key Steam
+        \key DistrictCooling
+        \key DistrictHeatingWater
+        \key DistrictHeatingSteam
```

Comment:
For `LifeCycleCost:UsePriceEscalation` and `LifeCycleCost:UseAdjustment`

Replace `Steam` with `DistrictHeatingSteam` like the rest of the objects

Reply:
OK, good catch.